runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/testing/TestDataflowRunner.java

```diff
@@ -77,9 +77,11 @@
   public static TestDataflowRunner fromOptions(
       PipelineOptions options) {
     TestDataflowPipelineOptions dataflowOptions = options.as(TestDataflowPipelineOptions.class);
-    dataflowOptions.setStagingLocation(Joiner.on("/").join(
+    String tempLocation = Joiner.on("/").join(
         new String[]{dataflowOptions.getTempRoot(),
-          dataflowOptions.getJobName(), "output", "results"}));
+        dataflowOptions.getJobName(), "output", "results"});
+    dataflowOptions.setTempLocation(tempLocation);
+    dataflowOptions.setStagingLocation(tempLocation);
```

Comment:
This to me says that we have something wrong in our initialization. Do we simply need to fix the Jenkins config?